.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
This is not a rename, just deleting one file and adding another file. git got this wrong here